package.json

```diff
@@ -36,7 +36,9 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "prepare": "husky install",
```

Comment:
I am not fimilur with this and all the sites about it are blocked.
This is what I found:
> Husky is a tool that allows us to work with Git hooks. Git hooks are scripts that you can set up to run at certain events in the Git lifecycle

Could you explain what this does a little more for me please?

Reply:
> I am not fimilur with this and all the sites about it are blocked. This is what I found:
> 
> > Husky is a tool that allows us to work with Git hooks. Git hooks are scripts that you can set up to run at certain events in the Git lifecycle
> 
> Could you explain what this does a little more for me please?

sure! 

**What are Git Hooks?**
[Git hooks](https://git-scm.com/docs/githooks) are scripts that you can set up to [run at certain events](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks) in the Git lifecycle. These events include different stages of a commit, like before a commit (pre-commit) and after a commit (post-commit).

These are useful in that they allow developers to run custom code tasks or even enforce standards by automating other scripts to run those tasks.

**What is Husky?**
[Husky](https://github.com/typicode/husky) is a tool that allows us to easily wrangle Git hooks and run the scripts we want at those stages.

It works by including an object right within our package.json file that configures Husky to run the scripts we specify. After that, Husky handles managing at which point in the Git lifecycle our scripts will run.


We are using husky here for:-  `format` the code when we run the `git commit` command

Reply:
> > We are using husky here for:- `format` the code when we run the `git commit` command
> 
> This seems like a great idea, so even though there are differences in each dev's formatting style, it's forced to an standard before commit, isn't it?

yea, using the `'prettier'` as the formatter

Reply:
I'm currently using [WebStorm](https://www.jetbrains.com/webstorm/) as an editor and it partially does what I think you are referring to for me. Then the other part, I'm a little familiar with from working on [toynet](https://toynet.projectreclass.org/). I think what your talking about are the the test and formating I follow from the  [contributing guide](https://docs.projectreclass.org/toynet/contributing-code-to-toynet/frontend-toynet-react) if I'm understand correctly. I'm sorry I'm a bit behind guys I've not even been coding for a year and python is my primary language but I'm a fast learner and have good intuition normally. My mentors are teaching me about all the other parts of programing.

Reply:
Oh and thank for the link. I browsed it briefly and bookmarked it for later. I want to get these figmas done and start working on that. I'll probably add it to my bookmarks repo, too.

Reply:
@ShashwatAgrawal20 @Ronnie018 @ashwaryy if your all okey with it, I will figure it out.